docs/basic_usage.md

```diff
@@ -335,3 +335,75 @@ TekHSI is compatible with PyVISA. You can mix PyVISA with TekHSI. This has some
     to take little or no time.
 3. TekHSI requires much less code that the normal processing of curve commands.
 4. The waveform output from TekHSI is easy to use with file readers/writers that allow this data to be quickly exported using the [tm_data_types](https://github.com/tektronix/tm_data_types) module.
+
+```python
```

Comment:
These scripts need to be actual python example files to enable linting.

Reply:
But similar to the other sections, team prefers to use this way.
As it is not intended to go as an example as of now.

Reply:
It is an example of basic usage. We can't lint this file in the markdown, so it needs to be python that is imported. That is our standard.

Reply:
I didn't realize that the other sections weren't python files. They need to be so that we can perform linting to ensure that we don't break example code.

Reply:
We want to lint everything, put them in a folder that indicates they aren't full examples, but we need to lint them if we want to ensure we catch breaking changes.